misaligned/misaligned.java

```diff
@@ -1,15 +1,24 @@
+package misaligned;
+
 public class Misaligned {
     static int printColorMap() {
         String majorColors[] = {"White", "Red", "Black", "Yellow", "Violet"};
         String minorColors[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
-        int i = 0, j = 0;
-        for(i = 0; i < 5; i++) {
-            for(j = 0; j < 5; j++) {
-                System.out.printf("%d | %s | %s\n", i * 5 + j, majorColors[i], minorColors[i]);
+        int majorColorIndex = 0, minorColorIndex = 0;
+        for(majorColorIndex = 0; majorColorIndex < 5; majorColorIndex++) {
+            for(minorColorIndex = 0; minorColorIndex < 5; minorColorIndex++) {
+            	assert(getPairNumberFromColor(majorColorIndex, minorColorIndex) == majorColorIndex * 5 + minorColorIndex);

```

Comment:
The assert is for testing, but is a part of production code. Try separating test code and production.